### DISCUSSION TITLE
Virtual touchscreen is not active.  Wait for Android to boot up.

### DISCUSSION THREAD

**Discussion by ochompsky** [tesla-android/issue-tracker#58] [Q&A] [answered]
No matter how many times I reinstall or reboot, I get this and cannot control Android.  I can see the Android device listed under Pihole so im pretty sure the connection between the two is fine.  After a few minutes, the message goes away on its own but I still have no control.

**Comment by galosre** [comment]
So I assume you pushed  the green plus and save it to be sure it is not that. Anyway, yesterday  I ran into this problem on my first try I did again the android auto installation for the rpi4 and at least this issue was gone.

**Comment by cukal** [comment, marked as answer]
Did you assign it ip 3.3.3.50 and hostname android? I forgot that step and had the same issue.

**Reply by ochompsky** [reply]
ugh im so dumb.  i was so sure i had input it, but i forgot to hit save.
